### DISCUSSION TITLE
Upcoming days view

### DISCUSSION THREAD

**Discussion by jedahan** [tenJirka/reGenda#7] [Ideas]
Something that would fill the screen with as many days needed depending on how busy/quiet the calendar is. Given a calendar which has one event on thursday, and one on sunday, something like:

```
# Thursday 19.10.2023
event a
# Friday
# Saturday
# Sunday 21.10
* event b
```

Or maybe even hiding friday and saturday

**Comment by tenJirka** [comment]
I like this idea and will implement it in reGenda, but it will take some time. I'm very busy right now with school and all the school projects.
